graph-builder/src/config/file.rs

```diff
@@ -41,7 +41,7 @@ impl FileOptions {
 
         let mut content = vec![];
         bufrd.read_to_end(&mut content)?;
-        let cfg = toml::from_slice(&content).context(format!(
+        let cfg = toml::from_str(std::str::from_utf8(&content)?).context(format!(
```

Comment:
@PratikMahajan Can you please add why these change are required because of the toml version change?

Reply:
dependency bump was a breaking change and required a corresponding code change as well